packages/react/src/Hint/Hint.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { render } from '@testing-library/react'
+import { createRef } from 'react'
+import { Hint } from '.'
+import '@testing-library/jest-dom'
+
+describe('Hint', () => {
+  it('renders', () => {
+    const { container } = render(<Hint />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<Hint />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('ams-hint')
+  })
+
+  it('renders an additional class name', () => {
+    const { container } = render(<Hint className="extra" />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('ams-hint extra')
+  })
+
+  it('supports ForwardRef in React', () => {
+    const ref = createRef<HTMLElement>()
+
+    const { container } = render(<Hint ref={ref} />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(ref.current).toBe(component)
+  })
+
+  it('renders the provided hint text', () => {
+    const { container } = render(<Hint hint="hint text" />)
+
+    const hint = container.querySelector('span.ams-hint')
+
+    expect(hint).toHaveTextContent('(hint text)')
+  })
```

Comment:
There seems to be a memory leak with the `Hint` component in Jest DOM.

We're getting the following error:

```
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
```

Running the tests with the `--logHeapUsage` option reveals that these four individual tests use over 4000MB(!) of RAM. The same goes for the four tests in `Label` and `FieldSet`.

@VincentSmedinga @alimpens @dlnr can you help me figure out what causes it?

Reply:
D'oh!